### PR TITLE
MLPAB-2386: Certificate provider opt out content updates

### PR DESCRIPTION
**Pull request by acsauk**
# Purpose

Fixes MLPAB-2386

### CODE REVIEW

File: web/template/certificateprovider/you_have_decided_not_to_be_a_certificate_provider.gohtml [hunk 1]
```diff
@@ -1,17 +1,18 @@
 {{ template "page" . }}
 
-{{ define "pageTitle" }}{{ tr .App "thankYou" }}{{ end }}
+{{ define "pageTitle" }}{{ trFormat .App "youHaveConfirmedYouDoNotWantToBeCertificateProvider" "DonorFullNamePossessive" (.App.Query.Get "donorFullNamePossessive") }}{{ end }}
```
Comment (hawx) [inline web/template/certificateprovider/you_have_decided_not_to_be_a_certificate_provider.gohtml:3]: this should not include their name, just be generic (I know some personalised ones have slipped in, but we need to remove them at some point)

Comment (acsauk) [reply web/template/certificateprovider/you_have_decided_not_to_be_a_certificate_provider.gohtml:3]: I'd forgotten we'd agreed that...I'll revert to thank you 👍

File: lang/en.json [hunk 2]
```diff
@@ -1114,7 +1114,8 @@
     "confirmYouDoNotWantToBeACertificateProvider": "Confirm you do not want to be the certificate provider",
     "confirmYouDoNotWantToBeACertificateProviderContent": "<p class=\"govuk-body\">You have told us that you do not want to be the certificate provide for {{ .DonorFullNamePossessive }} LPA :</p> <div class=\"govuk-inset-text\"> <p class=\"govuk-!-margin-bottom-1\"><span class=\"govuk-!-font-weight-bold\">LPA type:</span> {{ .LpaType }}</p> <p class=\"govuk-!-margin-bottom-1 govuk-!-margin-top-1\"><span class=\"govuk-!-font-weight-bold\">LPA number:</span> {{ .LpaUID }}</p> </div>",
     "whenYouSelectConfirmWeWillContactDonor": "When you select ‘Confirm’ we will contact {{ .DonorFirstNames }} to tell them what they should do next.",
-    "youHaveConfirmedYouDoNotWantToBeCertificateProvider": "You have confirmed that you do not want to be {{.DonorFullNamePossessive}} certificate provider.",
+    "youHaveConfirmedYouDoNotWantToBeACertificateProvider": "You have confirmed that you do not want to be a certificate provider.",
```
Comment (hawx) [inline lang/en.json:1117]: also no `.` at the end of the title

Comment (acsauk) [reply lang/en.json:1117]: Gah 🤦

**Review by hawx — APPROVED** [review]

**Pull request merged**